.changes/next-release/Bug Fix-7c49d65e-079c-4578-9c69-9dd7d4925b4e.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Redshift: 1. Remove the cache when the connection wizard is re-launched. 2. Modify the output message when a sql query is successful but doesn't return any record."
```

- Is this description meaningful for customers? 
    - "Remove the cache" means nothing for anyone not familiar with the implementation details.
    - The _Problem_ described in the PR is closer to something that is actually meaningful. It describes the actual behavior that is being fixed.
- Use multiple `npm run newChange` commands to describe multiple changes. There is no need to put everything in one item.


```suggestion
	"description": "Redshift: re-launched connection wizard may use the old (stale) connection."
```

---

Yes, I have made the suggested change.